t/test01.t

```diff
@@ -59,6 +59,11 @@ sub run_zonemaster_test_with_backend_API {
 	use_ok( 'Zonemaster::Backend::Config' );
 	my $config = Zonemaster::Backend::Config->load_config();
 
+	if ( not $ENV{ZONEMASTER_RECORD} ) {
+		Zonemaster::Engine->preload_cache( $datafile );
```

Comment:
Have you considered moving the call to reset() from run() to main() in zonemaster_backend_testagent instead of re-setting this stuff here?

Reply:
Well, the issue was that something needed to be reset after each call to the engine (sorry can't remember what it was). So the point to put it in run was to call it after each run.

Reply:
Well the reset was added because we thought it could be a reason for memory leaks. If it is not the case and if you think it raises other issues, feel free to remove it.

Reply:
@mattias-p I removed the call as in the TestAgent main method there is no dependency on the Engine and it seems to me cleaner not to add it for a call we basically not seem to need.